data/weilincheng.json

```diff
@@ -30,6 +30,14 @@
     }
   ],
   "milestones": [
+    {
+      "title": "Crossing the Threshold: Contributing to Open Source for the First Time",
+      "date": "March 2023",
+      "icon": "FaGithub",
+      "color": "blue",
+      "description": "Even if it's just a small enhancement, being able to contribute to the vibrant and diverse EiddeHub Community is an important milestone in my journey as a developer.",
```

Comment:
Hi @weilincheng, there is a minor typo in the spelling of **'EddieHub'**. Please correct it :+1:

Reply:
Thank you so much! I just corrected it.